### PR TITLE
Hypernode vpn documentation

### PR DESCRIPTION
**Pull request by Erik-Lamers1**
Add documentation on how to use the hypernode vpn feature.

### CODE REVIEW

File: Documentation/hypernode-api/settings/README.md [hunk 1]
```diff
@@ -27,3 +27,5 @@ this setting see [this changelog](https://support.hypernode.com/changelog/releas
 - `blackfire_enabled`: boolean - Indicates whether BlackFire is enabled on your node.
 - `blackfire_server_id`: - string - The BlackFire server id for your blackfire setup.
 - `blackfire_server_token`: string - The BlackFire server token for your blackfire setup.
+- `openvpn_enabled`: boolean - Enable or disable the [OpenVPN on Hypernode](/Documentation/hypernode-vpn/README.md)
```
Comment (kerrermanisNL) [inline Documentation/hypernode-api/settings/README.md]: You can leave the "the" out. As well as `on Hypernode` since these are the hypernode docs :p

Comment (vdloo) [reply Documentation/hypernode-api/settings/README.md]: done

File: Documentation/hypernode-vpn/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+### Using OpenVPN
+To be able to connect to your Hypernode you will need to install the latest version of the 
+[OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html).
+
+### Enable / Disable
+This can simply be done on the Hypernode itself.
+ ```
+ hypernode-systemctl settings openvpn_enabled --value <true/false>
+ ```
+This command will cause trigger the Hypernode automation to start installing OpenVPN and generate the configuration.
```
Comment (kerrermanisNL) [inline Documentation/hypernode-vpn/README.md:20]: *will trigger?

Comment (Erik-Lamers1) [reply Documentation/hypernode-vpn/README.md:20]: Fixed

**Pull request merged**